msteams-platform/apps-in-teams-meetings/enable-and-configure-your-app-for-teams-meetings.md

```diff
@@ -18,7 +18,7 @@ With apps for Teams meetings, you can expand the capabilities of your apps acros
 
 * Know how to develop Teams apps. For more information on how to develop Teams app, see [Teams app development](../overview.md).
 
-* Use your app that supports configurable tabs in the `groupchat` scope. For more information, see [group chat scope](../resources/schema/manifest-schema.md#configurabletabs) and [build a group tab](../build-your-first-app/build-channel-tab.md).
+* Use your app that supports configurable tabs in the groupchat scope. For more information, see [group chat scope](../resources/schema/manifest-schema.md#configurabletabs) and [build a group tab](../build-your-first-app/build-channel-tab.md).
```

Comment:
This needs an update for channel meetings. Below we say that that scenario requires "team" scope.

Reply:
@v-ksar please update " Use your app that supports configurable tabs in the groupchat scope." to " Use your app that supports configurable tabs in the groupchat and/or team scope."

Reply:
please update the "group chat scope" to link text to "scopes".  and update the "build a group tab" link text to "build your first tab app"

Reply:
Updated as mentioned.